### PR TITLE
Replace KUBEVIRT_WITH_CNAO to k8s-cnao-1.17 fake provider

### PR DESCRIPTION
**Pull request by qinqon**
The kubevirt automation expect just a target and parse this target to skip or focus tests, let's continue with this approach creating this fake provider, behind the scines is just the k8s-1.17 provider but we pass some flag to the infra scripts.

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: cluster-up/cluster/k8s-provider-common.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ function up() {
     prepare_config
 
     # Activate cluster-network-addons-operator if flag is passed
-    if [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
+    if [ "$WITH_CNAO" == "true" ]; then
```
Comment (phoracek) [inline cluster-up/cluster/k8s-provider-common.sh]: I recall Roman commenting on the benefit of keeping it namespaced.

File: cluster-up/cluster/k8s-1.17/README.md [hunk 2]
```diff
@@ -23,15 +23,6 @@ node01    NotReady   master    31s       v1.17.1
 node02    NotReady   <none>    5s        v1.17.1
 ```
 
-## Bringing the cluster up with cluster-network-addons-operator provisioned
```
Comment (phoracek) [inline cluster-up/cluster/k8s-1.17/README.md]: Why do you drop the documentation? I find it useful.

Comment (qinqon) [reply cluster-up/cluster/k8s-1.17/README.md]: I have move it.

File: cluster-up/cluster/k8s-provider-common.sh (hunk 1 above)
Comment (qinqon) [reply cluster-up/cluster/k8s-provider-common.sh]: This is kind of internal var, that I didn't namespaced it.

Comment (rmohr) [reply cluster-up/cluster/k8s-provider-common.sh]: It should probably default somewhere to false then. Strictly speaking. Not that we enforce that in general anywhere ;)

**Pull request closed without merging**